app.py

```diff
@@ -52,6 +53,8 @@
     corr_comparison_score.subscribe(weirdness_score)
     weirdness_score.subscribe(print)
 
+    influxdb_storage = flatliners.InfluxdbStorage()
```

Comment:
please wrap this in a `if` block that checks for the presence of `FLT_INFLUX_HOST` environment variable
we should keep the app always able to execute